src/aiortc/rtcrtpsender.py

```diff
@@ -221,6 +231,10 @@ async def _handle_rtcp_packet(self, packet):
                     else:
                         self.__rtt = RTT_ALPHA * self.__rtt + (1 - RTT_ALPHA) * rtt
 
+                # Stop writing playout delay when seen by receiver
+                if self.__playout_delay_write:
+                    self.__playout_delay_write = report.highest_sequence < self.__playout_delay_first_written
```

Comment:
This doesn't account for uint16 wrap-around, I think you need `uint16_gt(self.__playout_delay_first_written, self.__playout_delay_first_written)`